flink-cdc-common/src/main/java/org/apache/flink/cdc/common/event/FlushEvent.java

```diff
@@ -17,25 +17,49 @@
 
 package org.apache.flink.cdc.common.event;
 
+import java.util.Collections;
+import java.util.List;
 import java.util.Objects;
 
 /**
  * An {@link Event} from {@code SchemaOperator} to notify {@code DataSinkWriterOperator} that it
  * start flushing.
  */
 public class FlushEvent implements Event {
+    /** The schema changes from which table. */
```

Comment:
If I understand correctly, this `TableId` set is the "affected" tables that need to be flushed, not the "original table that initiates the schema change".